js/libs/builder.js

```diff
@@ -77,13 +77,19 @@ function generateFormDefaults(data) {
     name: '',
     dataSourceId: '',
     templateId: '',
+    previewingTemplate: '',
```

Comment:
what's this? @hcarneiro

Reply:
@squallstar That's a flag for when the user is previewing the template.